### ISSUE TITLE
Provide examples

### ISSUE THREAD

**Issue by boldtrn** [maplibre/maplibre-navigation-ios#35]
## Motivation

There should be some demo that people can get started with fast to have something they can work with.

## Implementation

I am not really sure what the issue is with SPM and examples, but maybe we need a separate repo for this? Or we add it is a subproject or something like this.

**Comment by michaelkirk** [comment]
This is something I'm interested in. 

Relatedly, for the sake of development/examples is there a way to get a test `Route` from fixture data - i.e. not fetched from a web service?

**Comment by boldtrn** [comment]
Yes, you can load a route from the fixtures, this is done in the tests already. For example see this test case here: https://github.com/maplibre/maplibre-navigation-ios/blob/main/MapboxCoreNavigationTests/RouteControllerTests.swift#L162
